server/publications/afOrders.js

```diff
@@ -14,7 +14,10 @@ Meteor.publish('afOrders', function () {
         'email': 1,
         'shopifyOrderNumber': 1,
         'shipping.address.phone': 1,
-        'history': 1
+        'history': 1,
```

Comment:
Going to add a sev3 card to review our publications and subscriptions at some point. We may do better with more varied publications in the future.